app/handlers/exceptions_app.rb

```diff
@@ -0,0 +1,43 @@
+class ExceptionsApp < Rambulance::ExceptionsApp
+  before_action :set_locals
+
+  def bad_request; end
+
+  def conflict; end
+
+  def internal_server_error
+    supportconfig = Supportconfig.new(dir: Rails.root.join('public'))
+    supportconfig.generate unless supportconfig.generated?
+    @supportconfig = '/' + supportconfig.output_filename
+  rescue StandardError => e
```

Comment:
```ruby
rescue StandardError, Cheetah::ExecutionFailed => e
```
missing the Cheetah exception (from `supportconfig.generate`)

Reply:
All exceptions inherit from StandardError